app/views/boards/show.html.erb

```diff
@@ -0,0 +1,28 @@
+<body>
+
+ <div class="container">
+
+  <div class="board">
+
+    <table class="one">
+      <% 4.times do %>
+     
+        <tr class="even">
+ 	       <% 8.times do %>
+ 	        <td></td>	 
+ 	       <% end %>
+ 	      </tr>
+  
+        <tr class="odd">
+         <% 8.times do %>
+ 	        <td></td>
+         <% end %>
+ 			  </tr>
+ 			
+      <% end %>
+
+	  </table>
+  </div>
+ </div>
```

Comment:
It looks like you're still having some issues with inconsistent indentation. Could you try adding `"translate_tabs_to_spaces": true` to your user-level Sublime settings? This should allow you to use the tab key everywhere (much easier than hitting space a bunch of times), while producing files that are always indented in terms of spaces.